### PR TITLE
[build] Remove PEAPI from library profiles

### PR DESCRIPTION
**Pull request by marek-safar**

### CODE REVIEW

**Comment by kumpera** [comment]
Won't this be a breaking change as we'll stop shipping it?

File: mcs/Makefile [hunk 1]
```diff
@@ -14,8 +14,8 @@ monotouch_runtime_SUBDIRS := build class
 monotouch_watch_runtime_SUBDIRS := build class
 monotouch_tv_runtime_SUBDIRS := build class
 xammac_SUBDIRS := build class
-aot_hybrid_SUBDIRS := build class ilasm
-aot_only_SUBDIRS := build class ilasm
```
Comment (akoeplinger) [inline mcs/Makefile]: this will break building the IL-based runtime tests

**Comment by marek-safar** [comment]
AFAIK we never ship these so better to remove it now that later

File: mcs/Makefile (hunk 1 above)
Comment (marek-safar) [inline mcs/Makefile]: I know, that's why it has label work-in-progress

**Pull request merged**